nix/scripts/shell.sh

```diff
@@ -41,16 +41,6 @@ nixArgs=(
     "--attr shells.${TARGET}"
 )
 
-config=''
-if [[ -n "${STATUS_GO_SRC_OVERRIDE}" ]]; then
-    config+="status-im.status-go.src-override=\"${STATUS_GO_SRC_OVERRIDE}\";"
-fi
-config+="status-im.build-type=\"${BUILD_TYPE}\";"
```

Comment:
@jakubgs, we had empty build type before refactoring.
Looks like it wasn't used for shell. FYI

Reply:
Good catch.